niceman/resource/docker_container.py

```diff
@@ -26,9 +30,12 @@ class DockerContainer(Resource):
     Environment manager which manages a Docker container.
     """
 
-    # Container properties
+    # Generic properties of any Resource
     name = attr.ib()
+
+    # Container properties
     id = attr.ib(default=None)
+    container_name = attr.ib(default=None)
```

Comment:
why do we need both `.name` and a `.container_name`?

Reply:
I thought decoupling the resource name from the Docker container name may be useful. Probably just complicates more than its utility. I will remove